src/controllers/update/update.filing.date.controller.ts

```diff
@@ -45,6 +47,1145 @@ export const post = async(req: Request, res: Response, next: NextFunction) => {
       }
       if (appData.update) {
         appData.update[FilingDateKey] = mapFieldsToDataObject(req.body, FilingDateKeys, InputDateKeys);
+        appData.update.review_trusts = [{
```

Comment:
The mock data will be removed once the PR is approved and the demo has been done to Matthew

Reply:
Since the data isn't needed for the PR, it would be good if you didn't add it to the commit at all. Saves cleaning it up after demo.

Reply:
Removed